GitHub-Test-4/Mulan/mulan_MatthewWilpula.md

```diff
@@ -0,0 +1,10 @@
+# Mulan Steps
+
+'[\D\d]+'
+    '<xml>\0</xml>'
+
+'(\n{2})(.+?)'
```

Comment:
@Matthew-W8 I have questions about this regex... what comes after the "don't-be-greedy" question mark?